core/zoe/docs/zoe.md

```diff
@@ -0,0 +1,263 @@
+# Zoe: Offer-Safety Enforcement
+
+Note: Zoe is currently at the pre-alpha stage. It has not yet been
+formally tested or hardened.
+
+This guide assumes some knowledge of the [ERTP fundamentals](./README.md).
+
+## What is Zoe? 
+
+__For users__: Zoe guarantees that as a user of a smart contract, you
+will either get what you wanted or get a refund back, even if the
+smart contract is buggy or backdoored. (In fact, the smart contract
```

Comment:
"backdoored" is a strange choice. I normally say "buggy or malicious". By "backdoored" I assume there is an illicit communications channel back to the attacker. Anything backdoored is malicious, but many malicious things are not backdoored. Confinement prevents backdoors but, obviously, not malice.

Reply:
good point